website/content/docs/nia/configuration.mdx

```diff
@@ -343,9 +357,19 @@ task {
 
 ### Task Source Input
 
-You can add an optional `source_input` block to the `task` block. The `source_input` block specifies a Consul object containing values or metadata to be provided to the Terraform Module. CTS supports the following source input types.
+You can optionally add one or more `source_input` blocks to the `task` block. A `source_input` block specifies a Consul object containing values or metadata to be provided to the Terraform Module. Both scheduled and dynamic tasks can be configured with `source_input` blocks.
+
+There are some limitations to the type of `source_input` blocks that can be configured for a task given the task's `condition` block and `services` field. This is because a task cannot have multiple configurations defining the same type of monitored variable:
```

Comment:
The limitations callout is important. Though reading this with fresh eyes, I think it would be useful to understand what source inputs are before getting to that level of detail, it might lose the reader. What do you think about these ideas:
1. Add a summary/skeleton of what portion of the configuration this section focuses on.
1. Adding a separator like a section title for limitations, like `Task Source Input Restrictions`
1. Move the limitation details after the available options (services and kv inputs).  This would be alluded to in the section summary and then linked.

Maybe a skeleton would be useful visualization to summarize what it means before starting the limitations. I'm hoping it would let the user know what portion of the config this section focuses on at a high level.

```hcl
task {
  name   = ""
  source = ""
  condition "<condition_type>" {
      // ...
  }
  source_input "<input_type>" {
      // ...
  }
}
```

```suggestion
You can optionally add one or more `source_input` blocks to the `task` block. A `source_input` block specifies a Consul object containing values or metadata to be provided to the Terraform Module. Both scheduled and dynamic tasks can be configured with `source_input` blocks.

#### Task Source Input Restrictions
There are some limitations to the type of `source_input` blocks that can be configured for a task given the task's `condition` block and `services` field. This is because a task cannot have multiple configurations defining the same type of monitored variable:
```

Reply:
Thanks for the suggestions! I think those make a lot of sense. Just updated with the suggestions - let me know if you have feedback or if I missed what you're saying. I added a warning notification around the 'allusion to the limitations' in the overview section - I wanted to be sure that readers who scan would notice the allusion now that the new restrictions section is at the "end". [preview link](https://consul-m581oyhy5-hashicorp.vercel.app/docs/nia/configuration#task-source-input)

Reply:
The yellow box looks good to catch my eye. The reorg is great, thanks!